.github/workflows/humble-binary-build.yml

```diff
@@ -5,9 +5,29 @@ on:
   pull_request:
     branches:
       - humble
+    paths:
+      - '**.hpp'
+      - '**.cpp'
+      - '.github/workflows/humble-binary-build.yml'
+      - '**/package.xml'
+      - '**/CMakeLists.txt'
+      - '**.xacro'
+      - '**.py'
+      - '**.yaml'
+      - 'ros2_control_demos-not-released.humble.repos'
```

Comment:
@christophfroehlich are these paths used to check for the changes in the files and then run the CI only if there are changes?

Reply:
exactly. if one changes only rst files, we don't run unnecessary CI checks.

Reply:
Awesome!

Reply:
@christophfroehlich Should we add these filters in other repos as well?. In case, we touch only the documentation, so it won't launch the unnecessary CI

Reply:
https://github.com/ros-controls/ros2_control/pull/1521
https://github.com/ros-controls/ros2_controllers/pull/1114
;)